requirements/dev.txt

```diff
@@ -1,8 +1,17 @@
 -r base.txt
 -r tests.txt
 
+# debugging
+ipdb==0.13.9
+ipython==8.1.1
```

Comment:
Can we please keep ipython in "base"? It's useful for administration of a running instance.

Reply:
Done.